scripts/core/editor3/reducers/suggestions.jsx

```diff
@@ -195,6 +195,47 @@ const createChangeBlockStyleSuggestion = (state, {blockType, data}) => {
     return saveEditorStatus(state, editorState, 'change-block-type', true);
 };
 
+/**
+ * Sanitize given content state on paste
+ *
+ * @param {ContentState} content
+ * @returns {ContentState}
+ */
+function sanitizeContent(content) {
+    let output = content;
+
+    const ignoreStyle = (style) => acceptedInlineStyles.indexOf(style) === -1;
+    const getSelection = (block, start, end) => SelectionState.createEmpty(block.getKey()).merge({
+        anchorOffset: start,
+        focusOffset: end,
+    });
+
+    content.getBlockMap().forEach((block) => {
+        // remove extra styles
+        block.findStyleRanges(
+            (character) => character.getStyle().some(ignoreStyle),
+            (start, end) => {
+                const selection = getSelection(block, start, end);
+                const inlineStyle = block.getInlineStyleAt(start).find(ignoreStyle);
+
+                output = Modifier.removeInlineStyle(output, selection, inlineStyle);
+            }
+        );
+
+        // remove any entities
+        block.findEntityRanges(
+            () => true,
+            (start, end) => {
+                const selection = getSelection(block, start, end);
+
+                output = Modifier.applyEntity(output, selection, null);
```

Comment:
Not sure if we have to delete the Entity, for example when you copy  a link from the same editor.

Reply:
yep we could keep that, I was testing with external links and those didn't work so I've rather disabled it for now